--- a/lib/vscode/build/gulpfile.reh.js
+++ b/lib/vscode/build/gulpfile.reh.js
@@ -341,9 +341,6 @@
 			.pipe(util.stripSourceMappingURL())
 			.pipe(jsFilter.restore);

-		const nodePath = `.build/node/v${nodeVersion}/${platform}-${arch}`;
-		const node = gulp.src(`${nodePath}/**`, { base: nodePath, dot: true });
-
 		let web = [];
 		if (type === 'reh-web') {
 			web = [
@@ -357,7 +354,6 @@
 			license,
 			sources,
 			deps,
-			node,
 			...web
 		);

@@ -468,7 +464,6 @@

 			const serverTaskCI = task.define(`vscode-${type}${dashed(platform)}${dashed(arch)}${dashed(minified)}-ci`, task.series(
 				compileNativeExtensionsBuildTask,
-				gulp.task(`node-${platform}-${arch}`),
 				util.rimraf(path.join(BUILD_ROOT, destinationFolderName)),
 				packageTask(type, platform, arch, sourceFolderName, destinationFolderName)
 			));
